doc/static/js/version_hacks.js

```diff
@@ -0,0 +1,137 @@
+function NCSVersionHacks () {
+  "use strict";
+
+  const LATEST_RELEASE_TITLE = "latest";
+
+  // XXX: do not remove the trailing '/'
+  const NCS_URL_ROOT = "https://developer.nordicsemi.com/nRF_Connect_SDK/doc/";
+  const NCS_URL_SUFFIX = "/nrf/index.html";
+  const LATEST_RELEASE_URL = NCS_URL_ROOT + "latest" + NCS_URL_SUFFIX;
+
+  //const VERSIONS_URL = NCS_URL_ROOT + "latest" + "/nrf/versions.json";
+  const VERSIONS_URL = "http://localhost:8000/versions.json";
+
+  var state = {
+    current_version: undefined,
+    version_data_url: VERSIONS_URL,
+  };
+
+  /*
+   * Infer the currently running version of the documentation
+   */
+  state.findCurrentVersion = function() {
+    // TODO: implement me!
+    this.current_version = "1.3.0"
+  };
+
+  /*
+   * This updates the list of versions displayed below the current version.
+   */
+  state.updateVersionDropDown = function() {
+    var VERSIONS = this.versions.VERSIONS;
+    var links = '';
+    for (var i = 0; i < VERSIONS.length; i++) {
+      var v = VERSIONS[i];
+      links += `<li><a href="${NCS_URL_ROOT}${v}${NCS_URL_SUFFIX}">${v}</a></li>\n`;
+    };
+
+    $("div.version").append(`
+  <span class="fa fa-caret-down dropit"></span>
+  <ul class="dropdown">
+    ${links}
+  </ul>`);
+
+    // hide version dropdown by default & toggle
+    $(".dropdown").hide();
+    $(".dropit").click(function(){
+      $(this).next(".dropdown").toggle();
+    });
+  };
+
+  /*
+   * Display a message at the top of the page to inform the user that the
+   * version currently being browsed is not the latest.
+   */
+  state.showVersion = function() {
+    var VERSIONS = this.versions.VERSIONS;
+
+    const OLD_RELEASE_MSG =
+      "You are looking at an older version of the documentation. You might " +
+      "want to switch to the documentation for the <a href='" +
+      LATEST_RELEASE_URL + "'>" + VERSIONS[0] + "</a> release or the " +
```

Comment:
"latest" (thus VERSIONS[0]) is actually the current state of development, and VERSIONS[1] is the latest release/tag.

Reply:
This naming confuses me!

Reply:
I can totally see why ... But releases are only once in a while. So "latest" is actually more current that "latest release".